sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -3417,12 +3409,12 @@ void archiveAllArchivableCases(int daysAfterCaseGetsArchived, LocalDate referenc
 		Root<Case> from = cq.from(Case.class);
 
 		Timestamp notChangedTimestamp = Timestamp.valueOf(notChangedSince.atStartOfDay());
-		cq.where(cb.equal(from.get(Case.ARCHIVED), false), cb.not(caseService.createChangeDateFilter(cb, from, notChangedTimestamp, true)));
+		cq.where(cb.equal(from.get(Case.ARCHIVED), false), cb.not(service.createChangeDateFilter(cb, from, notChangedTimestamp, true)));
 		cq.select(from.get(Case.UUID)).distinct(true);
 		List<String> caseUuids = em.createQuery(cq).getResultList();
 
 		if (!caseUuids.isEmpty()) {
-			archiveCoreEntities(caseUuids, null);
+			archive(caseUuids);
```

Comment:
If I understood it correctly, this PR also takes care of #7319, correct? In that case, I believe that we also need to specify an end of processing date for all entities that are automatically archived. @MartinWahnschaffe is that correct?

Reply:
Yes, this PR includes also #7319. Currently, Cases and Events have automatic archiving implemented.

Reply:
Okay, then this probably needs to be done for cases and events. Would still make sense to ask Martin for confirmation though. :)

Reply:
The archive process already does the calculation of the "end of processing date" for every core entity automatically archived.